benchmark/v5.2-performance-benchmarking-with-tpch.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: TiDB TPC-H 100 性能对比测试报告 - v5.1 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
+title: TiDB TPC-H 100 性能对比测试报告 - v5.2 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
```

Comment:
The alias is added in https://github.com/pingcap/docs-cn/pull/6868/files#diff-00428752dcd3c9acf5fe78f74e52359b2e723aeb5d03c91b87170bcdfd545469R3.